spec/requests/api/v1/participant_declarations_spec.rb

```diff
@@ -759,6 +759,63 @@
           expect(response.status).to eq 404
         end
       end
+
+      context "when querying a single old participant declaration owned by another provider" do
```

Comment:
could you add a spec for the voided endpoint (that it won't allow you to void a declaration from another provider) here and in other versions?

Reply:
added tests